app/src/main/java/org/mozilla/fenix/settings/TabsSettingsFragment.kt

```diff
@@ -40,9 +40,6 @@ class TabsSettingsFragment : PreferenceFragmentCompat() {
     private lateinit var inactiveTabsCategory: PreferenceCategory
     private lateinit var inactiveTabs: SwitchPreference
     private lateinit var searchTermTabGroups: SwitchPreference
-    private val shouldShowInactiveTabsTurnOffSurvey
-        get() = requireContext().settings().isTelemetryEnabled &&
-            requireContext().settings().shouldShowInactiveTabsTurnOffSurvey
```

Comment:
Rather than removing the code completely. I think we're only being asked to turn off the feature for the time being in the Jira ticket. So, we should be able to just set the default value of `shouldShowInactiveTabsTurnOffSurvey` to `false` for the time being.